_unittest/test_21_Circuit.py

```diff
@@ -923,6 +928,28 @@ def test_49_automatic_ami(self):
         )
         assert result
 
+    @pytest.mark.skipif(config["NonGraphical"] and is_linux, reason="Method not working in Linux and Non graphical.")
+    def test_49_automatic_ibis(self):
+        touchstone_file = os.path.join(local_path, "example_models", test_subfolder, touchstone_custom)
+        ibis_file = os.path.join(local_path, "example_models", "T15", "u26a_800_modified.ibs")
+        result, eye_curve_tx, eye_curve_rx = self.aedtapp.create_ibis_schematic_from_snp(
+            input_file=touchstone_file,
+            ibis_file=ibis_file,
+            component_name=None,
+            tx_buffer_name="DQ_FULL_800",
+            rx_buffer_name="DQ_FULL_800",
+            tx_pins=["A-MII-RXD1_30.SQFP28X28_208.P"],
+            rx_pins=["A-MII-RXD2_32.SQFP28X28_208.P"],
+            use_ibis_buffer=True,
+            differential=False,
+            bit_pattern="random_bit_count=2.5e3 random_seed=1",
+            unit_interval="31.25ps",
```

Comment:
@maxcapodi78 shouldn't we use 'units'?

Reply:
this is different. In IBIS simulation is unit_interval the distance between bits in seconds